<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCD</class>
 <widget class="QWidget" name="CCD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="CCD_connect">
       <property name="text">
        <string>Connect</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="CCD_trigger">
       <item>
        <property name="text">
         <string>Int.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ext.</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="CCD_exposure">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CCD_baseline">
       <property name="text">
        <string>baseline</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CCD_fastMode">
       <property name="text">
        <string>Fast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>D4Sigma:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="D4Sigma">
       <property name="toolTip">
        <string>fast diameter</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 18pt &quot;Noto Sans&quot;;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>val</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="CCD_plot"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="CCD_rec">
       <property name="text">
        <string>rec</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rec_limit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rec_counter">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filePath">
       <property name="text">
        <string>./test.dat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pathSelect">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
